app/src/main/java/org/oppia/app/player/exploration/ExplorationFragmentPresenter.kt

```diff
@@ -33,7 +50,24 @@ class ExplorationFragmentPresenter @Inject constructor(
     }
     return binding
   }
+  private fun getProfileData(): LiveData<StoryTextSize> {
+    return Transformations.map(
+      profileManagementController.getProfile(profileId),
+      ::processGetProfileResult
+    )
+  }
 
+  private fun subscribeToAudioLanguageLiveData() {
+    getProfileData().observe(activity, Observer<StoryTextSize> { result ->
+      FontScaleConfigurationUtil.adjustFontSize(activity,result)
+    })
+  }
+  private fun processGetProfileResult(profileResult: AsyncResult<Profile>): StoryTextSize {
```

Comment:
Ditto here & below. Please make sure all new methods have spacing to separate them.

Reply:
Done.